docs/csharp/tutorials/exploration/interpolated-strings.yml

```diff
@@ -79,7 +66,7 @@ items:
     In the previous step, two poorly formatted strings were inserted into the result string. One was a date and time value for which only the date was appropriate. The second was a price that didn't indicate its unit of currency. Both issues are easy to address. String interpolation lets you specify *format strings* that control the formatting of particular types. Modify the call to `Console.WriteLine` from the previous example to include the format strings for the date and price expressions as shown in the following line:
 
     ```csharp
-    Console.WriteLine($"On {date:d}, the price of {item} was {price:C2} per {unit}.");
+    Console.WriteLine($"On {date:d}, the price of {item.Name} was {item.Price} per {item.perPackage} items");
```

Comment:
The **C2** format string was omitted. On the other hand, it seems to use the invariant culture: 

`On 05/23/2019, the price of eggplant was ¤1.99 per 3 items`

Did you omit it deliberately because of that?

Reply:
I missed it. I did add the `C2` modifier.